lib/CardDAV/Plugin.php

```diff
@@ -12,6 +12,8 @@ function initialize(\Sabre\DAV\Server $server) {
         parent::initialize($server);
 
         $server->on('beforeUnbind', [$this, 'beforeUnbind']);
```

Comment:
Don't we need also to validate  `checkDomainMembersAccess` in `beforeUnbind` in order to to control domain member contact deletion too?